files/build_templates/init_cfg.json.j2

```diff
@@ -60,7 +60,11 @@
             "state": "{{state}}",
             "has_timer" : {{has_timer | lower()}},
             "has_global_scope": {% if feature + '.service' in installer_services.split(' ') %}true{% else %}false{% endif %},
+{%- if feature in ["lldp"] %}
+            "has_per_asic_scope": "{% if not DEVICE_RUNTIME_METADATA['ETHERNET_PORTS_PRESENT'] or ('CHASSIS_METADATA' in DEVICE_RUNTIME_METADATA and DEVICE_RUNTIME_METADATA['CHASSIS_METADATA']['module_type'] in ['supervisor']) %}False{% else %}True{% endif %}",
```

Comment:
this also requires  {% raw %} .... {% endraw %}

Reply:
Updated by @mlok-nokia